scripts/build/termux_step_setup_toolchain.sh

```diff
@@ -1,17 +1,23 @@
 termux_step_setup_toolchain() {
-	TERMUX_STANDALONE_TOOLCHAIN="$TERMUX_COMMON_CACHEDIR/android-r${TERMUX_NDK_VERSION}-api-${TERMUX_PKG_API_LEVEL}"
-	[ "$TERMUX_PKG_METAPACKAGE" = "true" ] && return
+	if [ "$TERMUX_PACKAGE_LIBRARY" = "bionic" ]; then
+		TERMUX_STANDALONE_TOOLCHAIN="$TERMUX_COMMON_CACHEDIR/android-r${TERMUX_NDK_VERSION}-api-${TERMUX_PKG_API_LEVEL}"
+		[ "$TERMUX_PKG_METAPACKAGE" = "true" ] && return
 
-	# Bump TERMUX_STANDALONE_TOOLCHAIN if a change is made in
-	# toolchain setup to ensure that everyone gets an updated
-	# toolchain
-	if [ "${TERMUX_NDK_VERSION}" = 25c ]; then
-		TERMUX_STANDALONE_TOOLCHAIN+="-v2"
-		termux_setup_toolchain_25c
-	elif [ "${TERMUX_NDK_VERSION}" = 23c ]; then
-		TERMUX_STANDALONE_TOOLCHAIN+="-v5"
-		termux_setup_toolchain_23c
-	else
-		termux_error_exit "We do not have a setup_toolchain function for NDK version $TERMUX_NDK_VERSION"
+		# Bump TERMUX_STANDALONE_TOOLCHAIN if a change is made in
+		# toolchain setup to ensure that everyone gets an updated
+		# toolchain
+		if [ "${TERMUX_NDK_VERSION}" = 25c ]; then
+			TERMUX_STANDALONE_TOOLCHAIN+="-v2"
+			termux_setup_toolchain_25c
+		elif [ "${TERMUX_NDK_VERSION}" = 23c ]; then
+			TERMUX_STANDALONE_TOOLCHAIN+="-v5"
+			termux_setup_toolchain_23c
+		else
+			termux_error_exit "We do not have a setup_toolchain function for NDK version $TERMUX_NDK_VERSION"
+		fi
+	elif [ "$TERMUX_PACKAGE_LIBRARY" = "glibc" ]; then
+		# What is TERMUX_STANDALONE_TOOLCHAIN???
```

Comment:
It's the path we put the toolchain in, doesn't seem like it is used here

Reply:
So I understand that in the case of compilation based on the glibc library, it is useless